src/libs/auth.js

```diff
@@ -19,29 +20,87 @@ import {
 import * as Utils from 'src/libs/utils'
 
 
+export const getOidcConfig = () => {
+  const metadata = {
+    authorization_endpoint: `${getConfig().orchestrationUrlRoot}/oauth2/authorize`,
+    token_endpoint: `${getConfig().orchestrationUrlRoot}/oauth2/token`,
+    ...(isGoogleAuthority() && { userinfo_endpoint: 'https://openidconnect.googleapis.com/v1/userinfo' })
+  }
+  return {
+    authority: `${getConfig().orchestrationUrlRoot}/oauth2/authorize`,
+    client_id: authStore.get().oidcConfig.clientId,
+    popup_redirect_uri: `${window.origin}/redirect-from-oauth`,
+    silent_redirect_uri: `${window.origin}/redirect-from-oauth-silent`,
+    metadata,
+    prompt: 'consent',
```

Comment:
What is the difference between 'consent' and 'login'?

Reply:
cc @nmalfroy as well if you have any insights

Reply:
Indeed, seems like for B2C
> Currently, the only valid value is `login`

Reply:
I left it as consent in B2C and it seems to behave ok.  I'm actually not sure that it makes much of a difference for B2C since we don't ask for sensitive scopes in B2C